workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -238,8 +276,7 @@ def register_parser(subparsers):
 
 
 def main(args):
-    diags = xr.open_dataset(args.input)
-    diags["time"] = diags.time - diags.time[0]
+    diags = _add_derived_diagnostics(xr.open_dataset(args.input))
```

Comment:
with this change, the `metrics.json` saved will include metrics computed from derived diagnostics, which is helpful if we want to use this in places other than the static HTML report.